tdrs-frontend/package.json

```diff
@@ -28,16 +21,15 @@
     "react-router-dom": "^5.2.0",
     "react-scripts": "^4.0.3",
     "redux": "^4.0.5",
-    "redux-oidc": "^4.0.0-beta1",
     "redux-thunk": "^2.3.0",
     "seamless-immutable": "^7.1.3",
-    "typescript": "^3.9.7",
     "uswds": "2.9.0"
   },
   "resolutions": {
     "lodash": "^4.17.20",
     "bl": "^4.0.3",
-    "node-forge": "0.10.0"
+    "node-forge": "0.10.0",
+    "ssri": "8.0.1"
```

Comment:
@jtwillis92 why did we need ssri v8?

`#9 1.411 warning Resolution field "ssri@8.0.1" is incompatible with requested version "ssri@^6.0.1"`

Reply:
It was needed to resolve a vulnerability found by Snyk - https://snyk.io/vuln/npm:ssri

We did confirm that the warning shown above does not prevent any package installation - its just there because the latest SSRI isn't explicitly marked as supported by one or more packages. Although it looks like that vulnerability is also fixed in version 6.0.2 which may satisfy this warning as well. I'll test that out locally and push up a change if so!

Reply:
It looks like a different package specifically requests `ssri@8.0.1` so we get a similar warning when rolling back to `6.0.2`:
```
warning Resolution field "ssri@6.0.2" is incompatible with requested version "ssri@^8.0.1"
```
In either case this warning is harmless as it doesn't prevent any package installation. Since we have to upgrade anyway to resolve that vulnerability and the latest version doesn't cause any issues it seems beneficial to go to the latest. We don't actually use this package directly anywhere - its just pulled in by other packages and this block is instructing npm/yarn to use that version for the dependencies it installs.

These warnings are also negligible:
```
warning "react-scripts > @typescript-eslint/eslint-plugin > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
warning " > @testing-library/user-event@7.2.1" has unmet peer dependency "@testing-library/dom@>=5".
warning " > jest-enzyme@7.1.2" has unmet peer dependency "jest@>=22.0.0".
warning "jest-enzyme > jest-environment-enzyme@7.1.2" has unmet peer dependency "jest@>=22.0.0".
```

Those only show because we are using react-scripts (CRA) packaged versions of the marked peer dependencies. Explicitly installing them causes errors in our implementation because we have not [ejected the project](https://create-react-app.dev/docs/available-scripts/#npm-run-eject).